culturefeed_mailing/culturefeed_mailing.module

```diff
@@ -334,6 +334,21 @@ function culturefeed_mailing_newsletter_block_form($form, &$form_state) {
 
 }
 
+/**
+ * Validate the email address using an external service.
+ *
+ * Implements hook_form_validate().
+ */
+function culturefeed_mailing_newsletter_block_form_validate($form, &$form_state) {
+  module_load_include('inc', 'culturefeed_data_validation', 'culturefeed_data_validation.helpers');
+  $datavalidationClient = _culturefeed_data_validation_get_datavalidation_client();
+
+  $response = $datavalidationClient->validateEmail($form_state['values']['mail']);
```

Comment:
try / catch should be added. User will receive a white screen is the api is down